.scalafmt.conf

```diff
@@ -1,4 +1,4 @@
-version = 3.7.11
```

Comment:
Most our projects use 3.7.11 and scalafmt-native on my Mac only works when you have a pre-built install that targets an exact version (eg 3.7.11). Long story short, it suits me to keep all the versions of scalafmt the same. 3.7.17 doesn't seem to have anything in it that justifies upgrading.

Reply:
I didn't know you were using `scalafmt-native`, there is also the standard scalafmt thats available (assuming you are using homebrew) which will work with any scalafmt version). Personally I haven't really noticed a speed difference locally with scalafmt vs scalafmt-native assuming you have one of the newer macs with mac sillicon.